ps3.py

```diff
@@ -0,0 +1,414 @@
+# 6.0001 Problem Set 3
+#
+# The 6.0001 Word Game
+# Created by: Kevin Luu <luuk> and Jenna Wiens <jwiens>
+#
+# Name          : Bogdan Zadorozhny
+# Collaborators : -
+# Time spent    : ~4 hours
+
+import math
+import random
+import string
+import re
+
+VOWELS = 'aeiou'
+CONSONANTS = 'bcdfghjklmnpqrstvwxyz'
+HAND_SIZE = 7
+
+SCRABBLE_LETTER_VALUES = {
+    'a': 1, 'b': 3, 'c': 3, 'd': 2, 'e': 1, 'f': 4, 'g': 2, 'h': 4, 'i': 1, 'j': 8, 'k': 5, 'l': 1, 'm': 3, 'n': 1,
+    'o': 1, 'p': 3, 'q': 10, 'r': 1, 's': 1, 't': 1, 'u': 1, 'v': 4, 'w': 4, 'x': 8, 'y': 4, 'z': 10
+}
+
+PATTERNS = {'word': "[a-zA-z*]+|!!", 'hands_number': "^[0-9]+$", 'replay_hand': "yes|no", 'letter': "[a-zA-z]",
+            'substitution': "yes|no"}
```

Comment:
there's no sense in creating a constant for that (or maybe it makes sense, but only for widely used cases, like 'word', 'letter', 'number', 'yes_no'). You can use appropriate pattern just in place where you need it. But if you decide to leave this dictionary - then please, convert it to Enum class